Dockerfile

```diff
@@ -7,50 +21,7 @@ RUN rm -rf /src/webconsole/dist
 RUN GOOS=linux GOARCH=amd64 WEBCONSOLE=default make immuadmin-static immudb-static
 RUN mkdir /empty
 
-FROM debian:bullseye-slim as bullseye-slim
-LABEL org.opencontainers.image.authors="Codenotary Inc. <info@codenotary.com>"
-
-COPY --from=build /src/immudb /usr/sbin/immudb
-COPY --from=build /src/immuadmin /usr/local/bin/immuadmin
-COPY --from=build "/etc/ssl/certs/ca-certificates.crt" "/etc/ssl/certs/ca-certificates.crt"
-
-ARG IMMU_UID="3322"
-ARG IMMU_GID="3322"
-
-ENV IMMUDB_HOME="/usr/share/immudb" \
-    IMMUDB_DIR="/var/lib/immudb" \
-    IMMUDB_ADDRESS="0.0.0.0" \
-    IMMUDB_PORT="3322" \
-    IMMUDB_PIDFILE="" \
-    IMMUDB_LOGFILE="" \
-    IMMUDB_MTLS="false" \
-    IMMUDB_AUTH="true" \
-    IMMUDB_DETACHED="false" \
-    IMMUDB_DEVMODE="true" \
-    IMMUDB_MAINTENANCE="false" \
-    IMMUDB_ADMIN_PASSWORD="immudb" \
-    IMMUDB_PGSQL_SERVER="true" \
-    IMMUADMIN_TOKENFILE="/var/lib/immudb/admin_token"
-
-RUN addgroup --system --gid $IMMU_GID immu && \
-    adduser --system --uid $IMMU_UID --no-create-home --ingroup immu immu && \
-    mkdir -p "$IMMUDB_HOME" && \
-    mkdir -p "$IMMUDB_DIR" && \
-    chown -R immu:immu "$IMMUDB_HOME" "$IMMUDB_DIR" && \
-    chmod -R 777 "$IMMUDB_HOME" "$IMMUDB_DIR" && \
-    chmod +x /usr/sbin/immudb /usr/local/bin/immuadmin
-
-EXPOSE 3322
-EXPOSE 9497
-EXPOSE 8080
-EXPOSE 5432
-
-HEALTHCHECK --interval=30s --timeout=30s --start-period=5s --retries=3 CMD [ "/usr/local/bin/immuadmin", "status" ]
-USER immu
-ENTRYPOINT ["/usr/sbin/immudb"]
-
-
-FROM scratch as scratch
+FROM scratch
```

Comment:
any reason we prefer scratch to gcr.io/distroless/static:nonroot?

Reply:
scratch is a bit smaller. But we should add a distroless Dockerfile within a Docker subfolder

Reply:
Ack, actually there is a non static version for distroless which has ca certs too, about the size overhead

```
gcr.io/distroless/static                                                       nonroot            2.36MB
gcr.io/distroless/base                                                         nonroot          20.3MB
```

Reply:
I built from both scratch and distroless (static and non static)

scratch vs distroless static ---> 1mb overhead
scratch vs distroless non static ---> 20mb overhead

Reply:
Better to keep scratch for now, as Dennis mentioned, we can add a separate distroless image in the build folder, Dockerfile.distroless